README.md

```diff
@@ -80,14 +109,35 @@ queue.async {
     This task begins execution once the above task suspends due to the one-second sleep.
     */
 }
-Task {
-    await queue.await {
-        /*
-        `async` context that can return a value or throw an error.
-        Executes after all other enqueued work has begun executing.
-        Work enqueued after this task will wait for this task to complete or suspend.
-        */
+await queue.await {
+    /*
+    `async` context that can return a value or throw an error.
+    Executes after all other enqueued work has begun executing.
+    Work enqueued after this task will wait for this task to complete or suspend.
+    */
+}
+```
+
+With an `ActorQueue` you can easily begin execution of asynchronous tasks from a nonisolated context in order:
+```swift
+func testActorQueueOrdering() async {
+    actor Counter {
+        func increment() -> Int {
+            count += 1
+            return count
+        }
+        var count = 0
+    }
+
+    let counter = Counter()
+    let queue = ActorQueue()
+    for iteration in 1...100 {
+        queue.async {
+            let incrementedCount = await counter.increment()
+            XCTAssertEqual(incrementedCount, iteration) // always succeeds
+        }
     }
+    await queue.await { }
 }
 ```
```

Comment:
Ideally we'd explain better how the `ActorQueue` and `FIFOQueue` differ in the above example test, but I'm wary of trying to communicate too many ideas at once.